### PR TITLE
new post: multi cluster monitoring with Prometheus and inlets

### PR DESCRIPTION
**Pull request by jsiebens**
Signed-off-by: Johan Siebens <johan.siebens@gmail.com>

### CODE REVIEW

**Comment by alexellis** [comment]
Can we have an image higher up in the post, like the architecture diagram we had before?

And a new lead image instead of the bridge.

**Comment by jsiebens** [comment]
The architecture diagram is already available in the introduction (line 16)

Do you have a source to find good lead images?

**Comment by jsiebens** [comment]
added new lead image

**Comment by alexellis** [comment]
I use unsplash and pexels - crop and then resize to 1660px width. And export to 70% quality aiming for < 280KB size.

**Comment by spekulatius** [comment]
Just jumping in with a suggestion for image optimization: 

Maybe https://github.com/marketplace/imgbot is a good idea for this. It reduces file sizes for images by removing human non-visible details from the images. Afaik it's free for public repos

File: blog/_posts/2020-12-12-multi-cluster-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: Multi-Cluster monitoring with Prometheus and inlets
+description: In many cases, a Kubernetes environment consist of multiple clusters. Learn how inlets can help setting up Prometheus Federation and bringing all the metrics in a single dashboard.
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud metrics prometheus federation helm
+author_img: jsiebens
+image: /images/2020-12-multi-cluster-monitoring/banner.jpg
+date: 12-12-2020
```
Comment (alexellis) [inline blog/_posts/2020-12-12-multi-cluster-monitoring.md:9]: Can you make the date Tuesday please?

File: blog/_posts/2020-12-12-multi-cluster-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: Multi-Cluster monitoring with Prometheus and inlets
```
Comment (alexellis) [inline blog/_posts/2020-12-12-multi-cluster-monitoring.md:3]: Multi-cluster monitoring with Prometheus and inlets -> Monitor multiple remote clusters with Prometheus and Grafana (second one is more use-case focused)

File: blog/_posts/2020-12-12-multi-cluster-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+description: In many cases, a Kubernetes environment consist of multiple clusters. Learn how inlets can help setting up Prometheus Federation and bringing all the metrics in a single dashboard.
```
Comment (alexellis) [inline blog/_posts/2020-12-12-multi-cluster-monitoring.md:4]: consist of -> consists of

File: blog/_posts/2020-12-12-multi-cluster-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud metrics prometheus federation helm
+author_img: jsiebens
+image: /images/2020-12-multi-cluster-monitoring/banner.jpg
+date: 12-12-2020
+---
+
+## Introduction
+
+More and more organisations are spreading their workloads across multiple Kubernetes cluster to improve availability and scalability. Perhaps you are running some batch jobs on Amazon Spot instances to reduce cost, while your heavy processing services are more likely benefit the power of bare-metal instances. From an operation point of view, it can become quite a challenge to monitor all those different clusters, running on various cloud providers or on-premises, all of them in an isolated network.
```
Comment (alexellis) [inline blog/_posts/2020-12-12-multi-cluster-monitoring.md:14]: The lead sentence has no source to back up the quote: "More and more organisations are spreading their workloads across multiple Kubernetes cluster"

How about?

There are two main reasons why you may have more than one Kubernetes cluster to monitor. The first use-case is where you own the clusters and have one per stage of development such as development, staging and production. Another is where you run a managed service, or have clients, who run workloads that you need to monitor for reliability, or for consumption as part of a service you run.

I.e. 

1 - internal use case. Different dev stages. Or different team/project clusters. For instance, a Monzo or a Form3.
2 - you are a managed service provider like a NetApp or a VSHN (on the homepage).

**Pull request merged**